README.md

```diff
@@ -30,7 +30,8 @@ FireCloud Orchestration Service
 * New paths to external endpoints should be added to `src/main/resources/configurations.conf`. Existing endpoint URLs are configured in `application.conf` and `test.conf`
 
 ## Building and Running
-See https://github.com/broadinstitute/firecloud-environment/blob/master/README.md for directions on running locally in docker.
+
+See https://github.com/broadinstitute/firecloud-develop for directions on running locally within Broad's DSDE environment.
```

Comment:
firecloud-develop tells me how to generate configs, but it tells me nothing about how to run orchestration. Somewhere, in some readme, we need a docker exec or docker run command.